### PR TITLE
new icon: figma (original, plain)

### PR DESCRIPTION
**Pull request by EnisMulic**

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -1082,6 +1082,22 @@
...
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#f24e1e",
```
Comment (Panquesito7) [inline devicon.json:1098]: There are no plain versions for this icon. Please add a font version for the icon (see the [Jupyter](https://github.com/devicons/devicon/tree/develop/icons/jupyter) icon for example).

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.json [hunk 2]
```diff
@@ -1061,6 +1061,22 @@
...
         ]
     },
+    {
+        "name": "figma",
+        "tags": [
+            "design"
+        ],
+        "versions": {
+            "svg": [
+                "original"
```
Comment (Panquesito7) [inline devicon.json:1071]:
```suggestion
                "original",
                "plain"
```

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! Figma will be included in the next release! 👍 🎉

**Comment by EnisMulic** [comment]
Can you also close #255

**Pull request merged**